### DISCUSSION TITLE
Focus element by pressing context menu button

### DISCUSSION THREAD

**Discussion by krupakov** [quasarframework/quasar#13288] [General - Components / Directives / etc] [answered]
Hello, I have a small problem.
I want to focus textarea by pressing a button.
It works perfectly fine with regular buttons, but I can't figure out how to make it work with context menu buttons.

Here's an example:
[https://codepen.io/krupakov/pen/ExQYVXb?editors=1010](https://codepen.io/krupakov/pen/ExQYVXb?editors=1010)

**Comment by MilosPaunovic** [comment, marked as answer]
What you were missing are the focus helpers for the `QMenu`:
[https://codepen.io/milospaunovic/pen/OJQLmer?editors=1010](https://codepen.io/milospaunovic/pen/OJQLmer?editors=1010)
